src/routes/Login/index.jsx

```diff
@@ -1,11 +1,64 @@
 import React from "react";
-import Navigation from "../../components/Navigation";
+import styles from "./login.module.css";
+
+import EmptySmallEllipseSvg from "../../../public/assets/Ellipse 22.svg";
+import SolidPinkEllipseSvg from "../../../public/assets/Ellipse 19 pink.svg";
+
+import Button from "../../components/Button";
+import Form from "./Form/Form.index";
+
+const {
+  container,
+  top,
+  EmptySmallEllipsePositionOne,
+  EmptySmallEllipsePositionTwo,
+  transparentSvg,
+  SolidPinkEllipsePosition,
+  header,
+  overlay,
+  loginHeader,
+  formMargin,
+  passcodeLink,
+  createAccountLink,
+} = styles;
 
 export default function Login() {
   return (
     <div>
```

Comment:
Because you already have container div, you actually don't need first div, because at that point he doing nothing except making it more nested.